### DISCUSSION TITLE
Sizce kaç dolar olur

### DISCUSSION THREAD

**Discussion by Alemci73** [IBAX-io/go-ibax#418] [Ideas]
IBAX

**Comment by Yenal4526** [comment]
1 dolar ile bir başlasın

**Comment by keremcanesa** [comment]
1 dolar ıyı olur

**Comment by girizgah** [comment]
çıktığı tarihten itibaren 1 yıl içeresinde kademeli olarak 1-3-8-13ü görebileceğine inanıyorum

**Comment by Mertdyr35** [comment]
2. Avalanche ağı yani Avax olur diyorum
